common/deities/01_iberic.txt

```diff
@@ -288,7 +320,7 @@ deity_nabia = { # ECONOMY
 	}
 	icon = deity_iberian_economy
 	passive_modifier = { global_commerce_modifier = deity_global_commerce_modifier_svalue }
-	omen = { naval_damage_done = omen_naval_damage_done_svalue } #Bonus from Omen
```

Comment:
these have changed in a lot of places. im not sure if we changed that to damage done on purpose or this is just an evolution of invictus's values. i think it's fine but just curious